.github/workflows/release.yaml

```diff
@@ -30,8 +30,10 @@ jobs:
         run: |
           make release
       - name: generate release notes
+        # Ignore failures for release-notes generation so they could still get
+        # generated manually before publishing.
         run: |
-          make generate-release-notes
+          make generate-release-notes || echo "Failed to generate release notes" >> _releasenotes/${{ env.RELEASE_TAG }}.md
```

Comment:
I assume this redirect only the echo into the file? (nothing from the make target)

Reply:
oh well, let me check this, good point!

Reply:
Yes, only redirects stdout from the echo!

Reply:
```sh
❯ (echo "hello" && fail) || echo "failed" >> out.txt
hello
zsh: command not found: fail

❯ cat out.txt
failed
```

Reply:
Thx for checking!